core/treepath/treepath_test.go

```diff
@@ -171,3 +171,21 @@ func TestApplyPath(t *testing.T) {
 		})
 	}
 }
+
+func TestString(t *testing.T){
```

Comment:
Typically way to format this would be with a table test: https://github.com/golang/go/wiki/TableDrivenTests

See above for some examples. Also, here's one:

```go
testCases := []struct{
  desc string
  in []int
  out string
}{
...
}
```